src/test/java/tictactoe/BoardConverterTest.java

```diff
@@ -1,12 +1,13 @@
 package tictactoe;
 
+import org.junit.Ignore;
```

Comment:
ctrl+alt+o seems to work! 👍 

I really need to deactivate those feature from IntelliJ 👎 :

- hyde the import
- display on only line some method